README.md

```diff
@@ -9,39 +9,51 @@ This repository contains the backend API for the Giraf Project. The API is a .ne
+
+2. Setup the connection to the local MySQL server.
+  - In the …\ web-api\GirafRest create a copy of the file “appsettings.template.json” and name it “appsettings.Development.json”.
+  - Open the created file “appsettings.Development.json” file with a text editor.
+  - Change the following (Remember to remove the “<” and “>”): 
+    * The DefaultConnection on line 3 making it use the previously setup database name and user, change to: "DefaultConnection": "server=localhost;port=3306;userid=user;password=password;database=giraf;Allow User Variables=True"
+    * The Jwt.JwtKey on line 24 to be any (random) string of, at least 40, alpha-numeric characters.
+    * The Jwt.JwtIssuer on line 25 to your name or organization. For example "Aalborg University"
+ 
+3. Open a terminal and navigate to …\ web-api\GirafRest folder.
+  - Run `dotnet restore`  
+  - Run `dotnet ef database update`
+  - Run `dotnet run --sample-data`
+
+The flags that can be used for the run:
 
         --prod=[true|false]		| If true then connect to production db, defaults to false.
-        --port=integer			| Specify which port to host the server on, defaults to 5000.
-        --list				    | List options
-        --sample-data		    | Tells the rest-api to generate some sample data. This only works on an empty database.
+        --port=integer		| Specify which port to host the server on, defaults to 5000.
+        --list			| List options
+        --sample-data		| Tells the rest-api to generate some sample data. This only works on an empty database.
         --logfile=string		| Toggles logging to a file, the string specifies the path to the file relative to the working directory.
 
-
-Once the API is running locally you can navigate to `http://localhost:5000/swagger/` to see and tryout requests to the endpoints. We recommend keeping a text file with often-used DTOs and bearer tokens as part of your workflow.
+Once the API is running locally you can navigate to http://localhost:5000/swagger/ to see and tryout requests to the endpoints. We recommend keeping a text file with often-used DTOs and bearer tokens as part of your workflow.
 
 3. (Optional) To login via swagger:
-  - Make a Account/Login request with valid login-info (username: `Tobias`, password: `password`)
+  - Make an Account/Login request with valid login-info (username: `Tobias`, password: `password`)
   - Copy the `data` field containing the token.
-  - Click on the green Authorize button (Or the padlocks)
+  - Click on the green Authorize button (Or the padlocks).
   - Write `bearer [your-token]` (note the space) in the input-field. 
   - Click Authorize and close the pop-up. 
-  - You are now authorized and can make autorized requests.
+  - You are now authorized and can make authorized requests.
+
+5. Let weekplanner use the local database and Giraf web_api server.
+  - In the weekplanner repository in the …/weekplanner/assets/environments.json file line 2 change the “http://srv.giraf.cs.aau.dk/DEV/API” to:
```

Comment:
We had an issue when doing this. If you use IntelliJ or such, the build file should be deleted and produced after this change has been made, otherwise it does not register the change to the enviroments.json file.